core-web/libs/portlets/dot-content-drive/portlet/src/lib/utils/functions.ts

```diff
@@ -1,5 +1,51 @@
-import { DotContentDriveFilters } from '../shared/models';
+import {
+    DotContentDriveDecodeFunction,
+    DotContentDriveFilters,
+    DotKnownContentDriveFilters
+} from '../shared/models';
 
+/**
+ * Decodes a multi-selector value.
+ *
+ * @param {string} value
+ * @return {*}  {string[]}
+ */
+const multiSelector: DotContentDriveDecodeFunction = (value: string): string[] =>
+    value
+        .split(',')
+        .map((v) => v.trim())
+        .filter((v) => v !== '');
```

Comment:
If value if undefined this will kill the app.

Reply:
I will fix this, this is also added on my [PR](https://github.com/dotCMS/core/pull/33013), kev needed it to work without problems

Reply:
Fixed! https://github.com/dotCMS/core/pull/33013/commits/c6a100099124d43d647016dc24bc84b6ddf9cd0c